core/common.go

```diff
@@ -0,0 +1,24 @@
+package core
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"io"
+)
+
+// RandomString and NewToken come from Boulder core/util.go
```

Comment:
Hah. Yes... I did feel guilty creating yet another `common.go`. I think the idea of using `token.go` in the WFE is the best bet. It felt like wfe.go was getting really large, splitting it up within the package achieves what I wanted. I switched both functions to be unexported as well.